.github/workflows/tests.yaml

```diff
@@ -204,54 +204,35 @@ jobs:
       if: always()
 
   file-ft-tests:
-    strategy:
-      matrix:
-        include:
-          - enable-null-compat: ''
-            job-name: file-ft-tests
-          - enable-null-compat: 'true'
-            job-name: file-ft-tests-null-compat
-    name: ${{ matrix.job-name }}
     runs-on: ubuntu-latest
     needs: build
     env:
       S3BACKEND: file
       S3VAULT: mem
       CLOUDSERVER_IMAGE: ghcr.io/${{ github.repository }}/cloudserver:${{ github.sha }}
       MPU_TESTING: "yes"
-      ENABLE_NULL_VERSION_COMPAT_MODE: "${{ matrix.enable-null-compat }}"
-      JOB_NAME: ${{ matrix.job-name }}
+      #ENABLE_NULL_VERSION_COMPAT_MODE: "${{ matrix.enable-null-compat }}"
```

Comment:
Depending on how atomic the commits that introduced these change son the usual branches are, maybe cherry-picking them will be easier for maintenance next time :slightly_smiling_face: